README.md

```diff
@@ -1191,9 +1191,22 @@ GET /apps/:guid?fields=expensive_field
 }
 ```
 
-### Proposal: Fields For Sub-Resources
+### Fields For Sub-Resources
+
+This is a mechanism for including multiple fields of related resources in a single response.
```

Comment:
The other interesting aspect of this is that the included fields potentially have different permissions than including the whole resource (field-level permissions). So I might be able to  `fields[related.resource]` even if I couldn't `includes=related.resource`. This also means that potentially I could `fields[related.resource]=foo` and not `fields[related.resource]=bar`, even if both of those would work for someone with more permissions (say, a global admin).

Reply:
Yes; that's the use-case we are aiming for. For example, domains are shared with organizations, "owners" of domains might have permission to read the org name, but not the rest of the Organization model.

In our initial implementation, `fields` will be documented and supported in a specific way: all users that can read the resource would be able to use the supported values for `fields`. For example: the only fields one can query on service instances is `fields[service_plan.service_offering]=name,guid`. If a user can list the service instances, they can read the service offering name and guid using (even without access to the service offering resource directly).